nipype/interfaces/freesurfer/preprocess.py

```diff
@@ -630,8 +630,26 @@ class ReconAllInputSpec(CommandLineInputSpec):
                            desc="Enable parallel execution")
     hires = traits.Bool(argstr="-hires", min_ver='6.0.0',
                         desc="Conform to minimum voxel size (for voxels < 1mm)")
+    mprage = traits.Bool(argstr='-mprage',
+                         desc=('Assume scan parameters are MGH MP-RAGE '
+                               'protocol, which produces darker gray matter'))
+    big_ventricles = traits.Bool(argstr='-bigventricles',
+                                 desc=('For use in subjects with enlarged '
+                                       'ventricles'))
+    brainstem = traits.Bool(argstr='-brainstem-structures',
+                            desc='Segment brainstem structures')
+    hippocampal_subfields_T1 = traits.Bool(
+        argstr='-hippocampal-subfields-T1', min_ver='6.0.0',
+        desc='segment hippocampal subfields using input T1 scan')
+    hippocampal_subfields_T2 = traits.Tuple(
+        File(exists=True), traits.Str(),
+        argstr='-hippocampal-subfields-T2 %s %s', min_ver='6.0.0',
+        desc=('segment hippocampal subfields using T2 scan, identified by '
+              'ID (may be combined with hippocampal_subfields_T1)'))
```

Comment:
No. A separate filename is [parsed](https://github.com/freesurfer/freesurfer/blob/dev/scripts/recon-all#L7443-L7454). I don't know enough about these processes to know if you'd use the same T2 for hippocampal subfields as for pial surface refinement, but certainly it's not required by their code that they be the same file.